packages/expo-modules-core/ios/ExpoModulesCore.h

```diff
@@ -1,5 +1,71 @@
+#import <ExpoModulesCore/EXBarcodeScannerProviderInterface.h>
+#import <ExpoModulesCore/EXBarometerInterface.h>
+#import <ExpoModulesCore/EXBridgeModule.h>
+#import <ExpoModulesCore/EXCameraInterface.h>
+#import <ExpoModulesCore/EXConstantsInterface.h>
+#import <ExpoModulesCore/EXDefines.h>
+#import <ExpoModulesCore/EXDeviceMotionInterface.h>
+#import <ExpoModulesCore/EXErrorCodes.h>
+#import <ExpoModulesCore/EXEventEmitter.h>
+#import <ExpoModulesCore/EXEventEmitterService.h>
+#import <ExpoModulesCore/EXExportedModule.h>
+#import <ExpoModulesCore/EXFaceDetectorManagerInterface.h>
+#import <ExpoModulesCore/EXFaceDetectorManagerProviderInterface.h>
+#import <ExpoModulesCore/EXFilePermissionModuleInterface.h>
+#import <ExpoModulesCore/EXFileSystemInterface.h>
+#import <ExpoModulesCore/EXFontManagerInterface.h>
+#import <ExpoModulesCore/EXFontProcessorInterface.h>
+#import <ExpoModulesCore/EXFontScalerInterface.h>
+#import <ExpoModulesCore/EXFontScalersManagerInterface.h>
+#import <ExpoModulesCore/EXGyroscopeInterface.h>
+#import <ExpoModulesCore/EXImageLoaderInterface.h>
+#import <ExpoModulesCore/EXInternalModule.h>
+#import <ExpoModulesCore/EXJavaScriptContextProvider.h>
+#import <ExpoModulesCore/EXKernelService.h>
+#import <ExpoModulesCore/EXLogHandler.h>
+#import <ExpoModulesCore/EXLogManager.h>
+#import <ExpoModulesCore/EXMagnetometerInterface.h>
+#import <ExpoModulesCore/EXMagnetometerUncalibratedInterface.h>
+#import <ExpoModulesCore/EXModuleRegistry.h>
+#import <ExpoModulesCore/EXModuleRegistryAdapter.h>
+#import <ExpoModulesCore/EXModuleRegistryConsumer.h>
+#import <ExpoModulesCore/EXModuleRegistryDelegate.h>
+#import <ExpoModulesCore/EXModuleRegistryHolderReactModule.h>
+#import <ExpoModulesCore/EXModuleRegistryProvider.h>
+#import <ExpoModulesCore/EXNativeModulesProxy.h>
+#import <ExpoModulesCore/EXPermissionsInterface.h>
+#import <ExpoModulesCore/EXPermissionsMethodsDelegate.h>
+#import <ExpoModulesCore/EXPermissionsService.h>
+#import <ExpoModulesCore/EXReactFontManager.h>
+#import <ExpoModulesCore/EXReactLogHandler.h>
+#import <ExpoModulesCore/EXReactNativeAdapter.h>
+#import <ExpoModulesCore/EXReactNativeEventEmitter.h>
+#import <ExpoModulesCore/EXReactNativeUserNotificationCenterProxy.h>
+#import <ExpoModulesCore/EXSingletonModule.h>
+#import <ExpoModulesCore/EXTaskConsumerInterface.h>
+#import <ExpoModulesCore/EXTaskInterface.h>
+#import <ExpoModulesCore/EXTaskLaunchReason.h>
+#import <ExpoModulesCore/EXTaskManagerInterface.h>
+#import <ExpoModulesCore/EXTaskServiceInterface.h>
+#import <ExpoModulesCore/EXUIManager.h>
+#import <ExpoModulesCore/EXUnimodulesCompat.h>
+#import <ExpoModulesCore/EXUserNotificationCenterProxyInterface.h>
+#import <ExpoModulesCore/EXUtilities.h>
+#import <ExpoModulesCore/EXUtilitiesInterface.h>
+#import <ExpoModulesCore/EXViewManager.h>
+#import <ExpoModulesCore/EXViewManagerAdapter.h>
+#import <ExpoModulesCore/EXViewManagerAdapterClassesRegistry.h>
+#import <ExpoModulesCore/ExpoModulesCore.h>
```

Comment:
Are you sure we need all of this? I think the same headers are already imported by `*-umbrella.h` 🤔 
Also, importing `ExpoModulesCore.h` seems to be circular 😅